raiden/storage/sqlite.py

```diff
@@ -174,7 +174,7 @@ def get_statechanges_by_identifier(self, from_identifier, to_identifier):
 
         return result
 
-    def get_events_by_identifier(self, from_identifier, to_identifier):
+    def get_events_by_identifier(self, from_identifier, to_identifier, limit = -1, offset = 0):
```

Comment:
@hackaugusto Please have a look at this. This still doesn't feel like a nice solution, but it gets the job done.